includes/class-amp-travel-taxonomies.php

```diff
@@ -0,0 +1,92 @@
+<?php
+/**
+ * Class for Travel taxonomies.
+ *
+ * @package WPAMPTheme
+ */
+
+/**
+ * Class Travel_Taxonomies
+ *
+ * @package WPAMPTheme
+ */
+class AMP_Travel_Taxonomies {
+
+	/**
+	 * AMP_Travel_Taxonomies constructor.
+	 */
+	public function __construct() {
+		add_action( 'init', array( $this, 'register_taxonomies' ) );
+	}
+
+	/**
+	 * Register 'activity' and 'location' taxonomy.
+	 */
+	public function register_taxonomies() {
+		register_taxonomy( 'activity', array( 'adventure', 'post' ), array(
```

Comment:
There's no action needed now. But when PR #33 is merged, this `adventure` string should probably be `AMP_Travel_CTP ::POST_TYPE_SLUG_SINGLE`.